brave-bom/pom.xml

```diff
@@ -134,6 +134,11 @@
         <artifactId>brave-context-rxjava2</artifactId>
         <version>${project.version}</version>
       </dependency>
+      <dependency>
+        <groupId>${project.groupId}</groupId>
+        <artifactId>brave-instrumentation-benchmarks</artifactId>
```

Comment:
I suspect it may be intentional that we don't include the benchmarks module in the bom. Is there a case where a third-party would depend on our benchmarks module?

Reply:
I don't know actually, I just added all of the projects. I'll leave this one up for regular contributers to decide.

Reply:
Personally I don't depend on it, but since it has its own package that's being published, I imagine getting the version alignment through the bom might be nice just in case.

Reply:
@JochemKuijpers I think it does not heart to have this modules here, as dependencies will have to be declared explicitly anyhow. I'm ok with this additions (also the tests module).